src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md

```diff
@@ -0,0 +1,404 @@
+---
+title: "Beautiful, efficient images for e‑commerce"
+author: samdutton
```

Comment:
```suggestion
authors:
  - samdutton
```

Reply:
https://github.com/GoogleChrome/web.dev/pull/856 switches up the syntax a bit so we can have more than one author on a post.